packages/mdc-tab-bar/adapter.js

```diff
@@ -25,7 +25,7 @@
 
 /* eslint-disable no-unused-vars */
 import {MDCTabDimensions} from '@material/tab/adapter';
-import {MDCTab} from '@material/tab/index';
+import {MDCTabFoundation} from '@material/tab/index';
```

Comment:
I would assume this is truly unused, even in annotations?

Reply:
Good catch